lib/srv/forward/sshserver.go

```diff
@@ -922,6 +922,11 @@ func (s *Server) dispatch(ctx context.Context, ch ssh.Channel, req *ssh.Request,
 			s.log.Debug(err)
 		}
 		return nil
+	case sshutils.PuTTYSimpleRequest:
+		// PuTTY automatically requests an additional 'simple' subsystem any time it connects to a server
+		// as a proxy and makes a subsystem request. As we don't support this request, we ignore it.
+		s.log.Debugf("%v: ignoring request for '%v' subsystem", s.Component(), sshutils.PuTTYSimpleRequest)
+		return nil
```

Comment:
I can't actually make PuTTY work with proxy forwarding mode at the moment because it doesn't use the regular Windows SSH agent, it uses its own implementation that seems to have different primitives. As such this may be redundant.